NewsApp/NewsGeneral/NewsViewController.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  NewsViewController.swift
+//  NewsApp
+//
+//  Created by Viki on 07/12/2023.
+//
+
+import UIKit
+import SnapKit
+
+class NewsViewController: UIViewController {
+    // MARK: - GUI Variables
+    
+    private lazy var scrollView: UIScrollView = {
+        let view = UIScrollView()
+        view.showsVerticalScrollIndicator = false
+        
+        return view
+    }()
+    
+    private lazy var contentView: UIView = UIView()
+    
+    private lazy var imageView: UIImageView = {
+        let view = UIImageView()
+        
+        view.image = UIImage(named: "image")
+        
```

Comment:
extra line
and i don't see this image in the simulator when i navigate to this screen